### ISSUE TITLE
XS_AUTHORIZATION_ADMIN authorization error

### ISSUE THREAD

**Issue by bhargavsrinadh** [SAP-samples/hana-shine-xsa#64]
We are able to launch the "launchpad" app but got the following error when tried to check the prerequisites. We are receiving this authorization error even though the role collection " XS_AUTHORIZATION_ADMIN" has been added to the user through XSA_ADMIN.

"An error occurred while checking your authorization. Please assign the XS_AUTHORIZATION_ADMIN role collection to your user and retry"

anyone facing this issue?

  - name: uaa
    properties:
      url: 'https://host:30032/uaa-security'
--

**Comment by apoeschl** [comment]
Same here. It seems not possible to create the role collections. When I look into the console, I got the following error messages:
![image](https://user-images.githubusercontent.com/1700188/77851251-528bf500-71d8-11ea-8fe3-04f911bdb55c.png)

Running on HANA2 SPS4 Rev. 45 and deployed the shine application via git.

**Comment by apoeschl** [comment]
I recognized that the build of the user-xsjs node was always creating a xsuaa service with a generated name, e.g. XSA_DEV-8qkhdn0gmoaxjl9o-hana-shine-xsa-shine-uaa and not using the shine-xsuaa I have created as described in the manual. Therefore I deleted this generated service instance and created a new one with the same name and uploaded the xs-security.json from the shine project when creating the service. This solved the issue for me.
